[chassis] test_bgp_shutdown test failing on the SONiC chassis

<!--
If you are reporting a new issue, make sure that we do not have any duplicates
already open. You can ensure this by searching the issue list for this
repository. If there is a duplicate, please close your issue and add a comment
to the existing issue instead.

If you suspect your issue is a bug, please edit your issue description to
include the BUG REPORT INFORMATION shown below. If you fail to provide this
information within 7 days, we cannot debug your issue and will close it. We
will, however, reopen it if you later provide the information.

For more information about reporting issues, see
https://github.com/Azure/SONiC/wiki#report-issues

---------------------------------------------------
GENERAL SUPPORT INFORMATION
---------------------------------------------------

The GitHub issue tracker is for bug reports and feature requests.
General support can be found at the following locations:

- SONiC Support Forums - https://groups.google.com/forum/#!forum/sonicproject

---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------
Use the commands below to provide key information from your environment:
You do NOT have to include this information if this is a FEATURE REQUEST
-->

**Description**
the `test_bgp_shutdown` fails on the chassis with following error
```
duthosts = [<MultiAsicSonicHost> str2-sonic-lc3-1, <MultiAsicSonicHost> str2-sonic-lc6-1, <MultiAsicSonicHost> str2-sonic-lc7-1, <MultiAsicSonicHost> str2-sonic-sup-1]
rand_one_dut_hostname = 'str2-sonic-sup-1'
def test_bgp_shutdown(duthosts, rand_one_dut_hostname):
duthost=duthosts[rand_one_dut_hostname]
BGP_DOWN_EXPECTED_LOG_MESSAGE = "admin state is set to 'down'"
BGP_DOWN_COMMAND = "config bgp shutdown all"
BGP_UP_COMMAND = "config bgp startup all"
>       check_syslog(duthost, "bgp_shutdown", BGP_DOWN_COMMAND, BGP_DOWN_EXPECTED_LOG_MESSAGE, BGP_UP_COMMAND)
BGP_DOWN_COMMAND = 'config bgp shutdown all'
BGP_DOWN_EXPECTED_LOG_MESSAGE = "admin state is set to 'down'"
BGP_UP_COMMAND = 'config bgp startup all'
duthost    = <MultiAsicSonicHost> str2-sonic-sup-1
duthosts   = [<MultiAsicSonicHost> str2-sonic-lc3-1, <MultiAsicSonicHost> str2-sonic-lc6-1, <MultiAsicSonicHost> str2-sonic-lc7-1, <MultiAsicSonicHost> str2-sonic-sup-1]
rand_one_dut_hostname = 'str2-sonic-sup-1'
log_fidelity/test_bgp_shutdown.py:36:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
duthost = <MultiAsicSonicHost> str2-sonic-sup-1, prefix = 'bgp_shutdown'
trigger_action = 'config bgp shutdown all'
expected_log = "admin state is set to 'down'"
restore_action = 'config bgp startup all'
def check_syslog(duthost, prefix, trigger_action, expected_log, restore_action):
loganalyzer = LogAnalyzer(ansible_host=duthost, marker_prefix=prefix)
loganalyzer.expect_regex=[expected_log]
try:
marker=loganalyzer.init()
duthost.command(trigger_action)
logger.info("Check for expected log {} in syslog".format(expected_log))
loganalyzer.analyze(marker)
except LogAnalyzerError as err:
logger.error("Unable to find expected log in syslog")
>           raise err
E           LogAnalyzerError: {'match_messages': {'/tmp/syslog.str2-sonic-sup-1.2022-01-25-19:24:56': []}, 'total': {'expected_match': 0, 'expected_missing_match': 1, 'match': 0}, 'match_files': {'/tmp/syslog.str2-sonic-sup-1.2022-01-25-19:24:56': {'expected_match': 0, 'match': 0}}, 'expect_messages': {'/tmp/syslog.str2-sonic-sup-1.2022-01-25-19:24:56': []}, 'unused_expected_regexp': ["admin state is set to 'down'"]}
duthost    = <MultiAsicSonicHost> str2-sonic-sup-1
err        = {'expect_messages': {'/tmp/syslog.str2-sonic-sup-1.2022-01-25-19:24:56': []},
...': 1, 'match': 0},
'unused_expected_regexp': ["admin state is set to 'down'"]}
expected_log = "admin state is set to 'down'"
loganalyzer = <tests.common.plugins.loganalyzer.loganalyzer.LogAnalyzer instance at 0x7f68e0ea50a0>
marker     = 'bgp_shutdown.2022-01-25-19:24:54'
prefix     = 'bgp_shutdown'
restore_action = 'config bgp startup all'
trigger_action = 'config bgp shutdown all'
log_fidelity/test_bgp_shutdown.py:24: LogAnalyzerError
```
<!--
Briefly describe the problem you are having in a few paragraphs.
-->

**Steps to reproduce the issue:**
1.
2.
3.

**Describe the results you received:**


**Describe the results you expected:**


**Additional information you deem important:**
<!--
software/ASIC/Hardware Flatform version and info
-->
    **Output of `show version`:**

    ```
    (paste your output here)
    ```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

--- a/tests/log_fidelity/test_bgp_shutdown.py
+++ b/tests/log_fidelity/test_bgp_shutdown.py
@@ -26,8 +26,8 @@ def check_syslog(duthost, prefix, trigger_action, expected_log, restore_action):
     finally:
         duthost.command(restore_action)
 
-def test_bgp_shutdown(duthosts, rand_one_dut_hostname):
-    duthost=duthosts[rand_one_dut_hostname]
+def test_bgp_shutdown(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
+    duthost=duthosts[enum_rand_one_per_hwsku_frontend_hostname]
 
     BGP_DOWN_EXPECTED_LOG_MESSAGE = "admin state is set to 'down'"
     BGP_DOWN_COMMAND = "config bgp shutdown all"

